packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -8139,6 +8139,306 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /eserviceTemplates/{templateId}/description:
+    parameters:
+      - name: templateId
+        in: path
+        description: The E-Service Template ID 
+        required: true
+        schema:
+          type: string
+          format: uuid
+    patch:
+      tags:
+        - eserviceTemplates
+      summary: Update the description of an E-Service Template
+      description: Update the description of an E-Service Template
+      operationId: updateEServiceTemplateDescription
```

Consistency with the corresponding e-services endpoint `updatePublishedEServiceDescription`

```suggestion
      summary: Update the description of an E-Service Template after its publication
      description: Update the description of an E-Service Template in Published or Suspended state
      operationId: updatePublishedEServiceTemplateDescription
```